### DISCUSSION TITLE
Mind to share your favorite proverbs

### DISCUSSION THREAD

**Discussion by aminokay** [NAIST-SE/AttractingContributors#930] [2023 Attraction Contributions]
It is good to be reminded by simple saying or PROVERBS that we wish someone would say to us. So why not share them it [here](https://github.com/aminokay/AttractingContributors-proverbs.git) and let others see and wish it is going to have a good impact to our life journey. let start with my favourite one :
```
" if you going to be fast, go alone,
     if you going to be far, go together " - african provebs
```

**Comment by bconfessor** [comment]
Done! Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md
